NAMESPACE

```diff
@@ -96,6 +98,7 @@ export(until_)
 export(wait)
 export(wait_)
 export(write_cert)
+importFrom(later,later)
```

Comment:
This is needed just to force the later package to load before nanonext does; there's a `R_RegisterCCallable`/`R_GetCCallable` dance that needs to happen, otherwise you get an error when the `later_api.h` static initializer loads.